package.json

```
@@ -37,8 +37,8 @@
     "ts-node": "^10.1.0"
   },
   "scripts": {
-    "prebuild:plugins": "npm --prefix plugins install",
-    "build:plugins": "npm --prefix plugins run build",
+    "build:plugins": "NODE_ENV=production && node build-plugins.js",
+    "build:plugins:dev": "node build-plugins.js",
```

Would it be possible to merge these two into one?

---

yes, can remove `NODE_ENV=production` and it will work based on the `NODE_ENV` set
```suggestion
    "build:plugins": "node build-plugins.js",
```